scss/elements/dialogs.scss

```diff
@@ -12,4 +9,10 @@
   > :last-child {
     margin-bottom: 0;
   }
+
+  &.is-rounded {
+    color: $base-color;
+    border: none;
+    box-shadow: 4px 0, -4px 0, 0 4px, 0 -4px;
```

Comment:
Are we sure that we always want the border to match the color of the text? I could easily see someone wanting different colored text within a dialog while maintaining a black border. With the "default color" solution of box shadows this is not possible. That's why I did not make this change when updating most elements to use the default color mentioned in issue #60

Reply:
@jjspace: Yeah, I agree with you. However, this is the way that we handle it in a few different spots in the framework. I think it’s fine for now, but we should absolutely investigate alternatives to the box-shadow method we’re currently using for borders.

Reply:
Right, I understand that but I checked all the places that `box-shadow` is used without colors specified and all but one instance are changes that I made in PR #143 where I took special care not to change it in places that could potentially have text in them for this very reason. The only other spot that uses the `color` property for the `box-shadow` is in `.nes-progress` class which will never have text inside so it shouldn't be an issue.
IMHO if the aim is to stick to what we currently do I think the colors should be defined in the `box-shadow` and not rely on `color` for the `<dialog />` component which will almost always have text inside it.

Reply:
Yeah @jjspace, I understand what you're saying now. I've fixed the issue. 😉